3회차/우상민/0_민석이의과제체크하기.py

```diff
@@ -1,3 +1,13 @@
 import sys
 
 sys.stdin = open("_민석이의과제체크하기.txt")
+T = int(input())
+for C in range(1, T+1):
+    student = []
+    N, M = map(int, (input().split()))
+    X = input().split()
+    for i in range(1, N+1):
+        student.append(str(i))
+    answer = set(student) - set(X)
+    A = sorted(map(int, (answer)))
+    print(f'#{C}', *A)
```

Comment:
좋은 풀이네요 👍 
사실 출력할 때 꼭 `map`으로 `int`를 해줄 필요는 없어요!
```suggestion
    A = sorted(answer)
    print(f'#{C}', *A)
```